mono/mini/debugger-engine.c

```diff
@@ -737,26 +759,23 @@ mono_de_ss_req_release (SingleStepReq *req)
 	dbg_lock ();
 	g_assert (req->refcount);
 	req->refcount --;
-	if (req->refcount == 0)
+	if (req->refcount <= 0)
```

Comment:
What does it mean if `refcount` was 0 before the decrement here?  how does that happen?

Reply:
This is wrong, I changed while I was doing some tests and didn't remove. I will undo this change.